src/net/sourceforge/kolmafia/persistence/FaxBotDatabase.java

```diff
@@ -121,9 +122,23 @@ public static final FaxBot getFaxbot(final String botName) {
     return null;
   }
 
-  public static final String botName(final int i) {
-    FaxBot bot = FaxBotDatabase.getFaxbot(i);
-    return bot == null ? null : bot.name;
+  public static final List<FaxBot> getSortedFaxbots() {
+    // Get preferred faxbot or null
+    FaxBot preferred = getFaxbot(Preferences.getString("lastSuccessfulFaxbot"));
+    // Use original list
```

Comment:
Given that "preferred" faxbot only applies to the one selected when the GUI is opened, I'd be fine with changing the comment and the name of the variable.  NitPickers R Us.